images/inventory-generator/root/usr/local/bin/entrypoint

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
```

Comment:
Suggestion: set shell options in the script, not in the shebang -- if the script is executed like `/path/to/shell path/to/script`, the `-e` flag is suddenly gone.

Also, I recommend:

```bash
set -o errexit
set -o nounset
set -o pipefail
```

This is what we do most often (see shell scripts in Origin), each option for a good reason.